util/reggen/countermeasure.py

```diff
@@ -34,6 +37,8 @@
     'SPARSE',
     'DIFF',
     'REDUN',
+    'REGWEN',
+    'REGREN',
```

Comment:
I have added `REGREN` so that we can name the reverse direction (for reads) as well.
We don't have the same level of `reggen` support for this, but there are instances where we implement this with custom logic, e.g. for the OTP SW_CFG partitions.

Reply:
sounds good to me.